qt/python/mantidqt/widgets/sliceviewer/view.py

```diff
@@ -490,11 +493,36 @@ def set_normalization(self, ws, **kwargs):
             self.presenter.normalization = mantid.api.MDNormalization.NoNormalization
             self.norm_opts.setCurrentIndex(0)
 
+    def get_default_scale_norm(self):
```

Comment:
This is really good now and works well. Thank you for making all of the changes.

I should have spotted this before but all of the other keys in the settings use PascalCase. Could we use that here too, e.g. `SliceViewer/` and could we pull them out as constants and the top of the module so we don't repeat them?

Sorry for not catching this sooner.

Reply:
To clarify, should it be the whole key in pascal case or just the section name?

Reply:
Sorry I didn't see this sooner. Just the primary key is fine. It looks like we are not too consistent on this. I've just fixed a minor typo in the name for the power key.

This all looks great now.